app/views/layouts/_dhtmlxlayout_explorer.html.haml

```diff
@@ -77,14 +62,15 @@
     #{layoutb_name}.cells("b").attachObject('#{right_cell_middle}');
     #{layoutb_name}.cells("c").attachObject('#{right_cell_bottom}');
 
-    #{layout_name}.attachEvent('onPanelResizeFinish', miqResizeTaskbarCell);
-    #{layout_name}.attachEvent('onResizeFinish', miqResizeTaskbarCell);
+    // TODO
+    #{layoutb_name}.attachEvent('onPanelResizeFinish', miqResizeTaskbarCell);
+    #{layoutb_name}.attachEvent('onResizeFinish', miqResizeTaskbarCell);
 
     - if left_cell_bottom
-      #{layout_name}.cells("c").setHeight(32);
-      #{layout_name}.cells("c").hideHeader();
-      #{layout_name}.setAutoSize("b", "a;b");
-      #{layout_name}.cells("c").attachObject('#{left_cell_bottom}');
+      #{layout_name}.cells("b").setHeight(32);
+      #{layout_name}.cells("b").hideHeader();
+      // #{layout_name}.setAutoSize("b", "a;b"); TODO: analyze this
```

Comment:
The whole comment here should probably be removed (unless this still is a work in progress)